templates/help/faq.html.twig

```diff
@@ -10,6 +10,8 @@
         <h2><i class="fa fa-question-circle-o" aria-hidden="true"></i> Technical FAQs</h2>
     </div>
     <a href="{{ faqs.web_link }}" target="_blank" class="btn btn-primary"><i class="fa fa-external-link" aria-hidden="true"></i> Click here to visit the HealthPro FAQ document</a>
-    {% include 'help/footer.html.twig' %}
+<footer class="footer help-footer">
```

Comment:
I would suggest keeping the `footer.html.twig` file for all the places that still need the footer, but extracting the `helptext.html.twig`, as you've done. The footer template would load the helptext template, so it's a little bit nested, but avoids duplication.

Reply:
Awesome Thanks for the feedback, I'll make those changes.